src/index.html

```diff
@@ -6,9 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo"
+        class="logo__image">
```

Comment:
Need to check and fix it everywhere
<img width="1051" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/5cd9d37b-36bf-4877-b9f4-0cb72e18c369">